python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -1931,10 +2002,19 @@ def _remeasure_star_background(self, result, background_to_photometric_ratio=Non
             background=result.background,
             backgroundToPhotometricRatio=background_to_photometric_ratio,
         ).background
-        # Isolate the final pedestal background to log the computed value
+        # Isolate the final pedestal background to log the computed value. Set
+        # a maximum allowable to avoid unnaturally high values (this step is
+        # largely trying to correct for oversubtraction).
         pedestalBackground = afwMath.BackgroundList()
         pedestalBackground.append(pedestalBackgroundList[1])
         pedestalBackgroundLevel = pedestalBackground.getImage().array[0, 0]
+        if pedestalBackgroundLevel > self.config.star_background_max_pedestal:
```

Comment:
I just realized this is wrong ... this needs to be an absolute value, and the sign preserved if you're setting to the max.

Reply:
Okay, then please add an opposite polarity (as we discussed).  And these should probably be set to some fraction of the median background rather than a fixed constant.